src/simulation/hdf_simulation_store.jl

```diff
@@ -104,22 +108,32 @@ df = PowerSimulations.read_result(DataFrame, store, model, :variables, var_name,
 function open_store(
     ::Type{HdfSimulationStore},
     directory::AbstractString,
-    mode = "r",
+    mode = "r";
     filename = HDF_FILENAME,
+    problem_path = nothing,
 )
-    return HdfSimulationStore(joinpath(directory, filename), mode)
+    return HdfSimulationStore(
+        joinpath(directory, filename),
+        mode,
+        problem_path = problem_path,
```

Comment:
Should the path be the same as the directory? I don't understand why we pass 2 different paths.